compiler/tests/powdr_std.rs

```diff
@@ -40,7 +40,7 @@ fn gen_halo2_proof(file_name: &str, inputs: Vec<Bn254Field>) {
         inputs,
         &mktemp::Temp::new_dir().unwrap(),
         true,
-        Some(backend::BackendType::Halo2),
+        Some(backend::BackendType::Halo2Mock),
```

Comment:
Otherwise, the test takes very long and [eventually gets cancelled](https://github.com/powdr-labs/powdr/actions/runs/6665784403/job/18116080314?pr=716).

I think it's completely fine to use the mock prover here, as it checks that all constraints are satisfied and that's all we want to test here.